Components/Note.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { View, StyleSheet, TextInput, Button, Text, TouchableOpacity, Alert } from 'react-native'
+import { Header, CheckBox, } from 'react-native-elements';
+import InputNote from 'ToDoProject/Components/InputNote';
+import { setDoneTask, onDeleteTask } from 'ToDoProject/Actions/ActionCreator';
+import {connect} from 'react-redux';
+
+
+class Note extends Component {
+ 
+    render() {
+        return (
+            <View style={{ flex: 1, flexDirection: "row", justifyContent: 'space-between', marginBottom: 15 }}>
+
+
+                <View style={{ flex: 6 }}>
+                    <CheckBox
+                        title={this.props.text}
+                        checked={this.props.isDone}
+                        onPress={() => this.props.setDoneTask(this.props.index)}
+                    />
+                    {/* Alert.alert('You need to...') */}
+                    {/* <Text>asdasd{this.props.text}</Text> */}
+                </View>
+
+                <View style={{ flex: 1, flexDirection: "column" }}>
+                    <TouchableOpacity
+
+                        style={{ flex: 1, justifyContent: "center", backgroundColor: this.props.isDone ? 'green' : 'red', borderRadius: 10 }}
```

Comment:
for long list of properties will be more readable place each property in new line

Reply:
I plan to make a stylesheet for everyone in next commit